Code/Engine/RendererVulkan/Shader/Implementation/ShaderVulkan.cpp

```diff
@@ -56,93 +56,69 @@ void ezGALShaderVulkan::SetDebugName(const char* szName) const
 
 ezResult ezGALShaderVulkan::InitPlatform(ezGALDevice* pDevice)
 {
-  ezGALDeviceVulkan* pDXDevice = static_cast<ezGALDeviceVulkan*>(pDevice);
-
-  // TODO
-#if 0
-  ID3D11Device* pD3D11Device = pDXDevice->GetDXDevice();
+  ezGALDeviceVulkan* pVulkanDevice = static_cast<ezGALDeviceVulkan*>(pDevice);
+  vk::ShaderModuleCreateInfo createInfo;
 
   if (m_Description.HasByteCodeForStage(ezGALShaderStage::VertexShader))
   {
-    if (FAILED(pD3D11Device->CreateVertexShader(m_Description.m_ByteCodes[ezGALShaderStage::VertexShader]->GetByteCode(),
-                                                m_Description.m_ByteCodes[ezGALShaderStage::VertexShader]->GetSize(), nullptr,
-                                                &m_pVertexShader)))
-    {
-      ezLog::Error("Couldn't create native vertex shader from bytecode!");
-      return EZ_FAILURE;
-    }
+    createInfo.codeSize = m_Description.m_ByteCodes[ezGALShaderStage::VertexShader]->GetSize();
```

Comment:
Make that a loop to reduce code duplication?

Reply:
Will be part of my next PR as I have to rewrite the class anyways.